uiplib/utils.py

```diff
@@ -17,6 +19,20 @@ def make_dir(dirpath):
     if sys.platform.startswith('linux'):
         os.chmod(dirpath, 0o777)
 
+
+def exit_UIP(service=False):
+    pid_file = os.path.join(HOME_DIR, 'daemon-uip.pid')
+    if service:
```

Comment:
well i didnt want this variable since if UIP gets exited from somewhere else, we'd still want to kill the pid process right?